roles/ceph-rgw/defaults/main.yml

```diff
@@ -48,6 +48,7 @@ rgw_pull_proto: "http"
 ##########
 
 ceph_rgw_civetweb_port: "{{ radosgw_civetweb_port }}"
-ceph_rgw_docker_extra_env: -e CLUSTER={{ cluster }} -e RGW_CIVETWEB_PORT={{ ceph_rgw_civetweb_port }}
+ceph_rgw_civetweb_ip: "{{ radosgw_civetweb_bind_ip }}"
+ceph_rgw_docker_extra_env: -e CLUSTER={{ cluster }} -e RGW_CIVETWEB_PORT={{ ceph_rgw_civetweb_port }} -e RGW_CIVETWEB_IP {{ ceph_rgw_civetweb_ip }}
```

Comment:
This should be `RGW_CIVETWEB_IP={{ radosgw_civetweb_bind_ip }}`, you forgot the `=`. There is no need to add an extra variable.